modules/RTC/CodecSelection.js

```diff
@@ -85,8 +75,16 @@ export class CodecSelection {
      * @private
      */
     _isCodecSupported(preferredCodec) {
+        if (!preferredCodec) {
+            return false;
+        }
+
+        if (preferredCodec === CodecMimeType.VP9 && !this.enforcePreferredCodec && !browser.supportsVP9()) {
```

Comment:
Why do we check the enforcement flag for this codec specifically?

Reply:
This is because our client doesn't support VP9 on Firefox/Safari though the browser supports it. There are folks in the community that use this flag to enforce VP9 selection on those browsers.